ci/jenkins/test-vmc.sh

```diff
@@ -478,20 +478,11 @@ function run_integration {
     VM_IP=$(govc vm.ip ${VM_NAME}) # wait for VM to be on
 
     set -x
-    if [[ ${flag} == "multicluster" ]];then
-      echo "===== Run Multi-cluster Integration tests ====="
-      # umask ensures that files are cloned with the correct permissions so that Docker caching can be leveraged
-      ${SSH_WITH_UTILS_KEY} -n jenkins@${VM_IP} "PATH=$PATH:/usr/local/go/bin && umask 0022 && git clone ${ghprbAuthorRepoGitUrl} antrea && cd antrea && git checkout ${GIT_BRANCH} && cd multicluster && NO_LOCAL=true make test-integration"
-      if [[ "$COVERAGE" == true ]]; then
-        run_codecov "mc-integration-tests" "coverage-integration.txt" "" true ${VM_IP}
-      fi
-    else
-      echo "===== Run Integration tests ====="
-      # umask ensures that files are cloned with the correct permissions so that Docker caching can be leveraged
-      ${SSH_WITH_UTILS_KEY} -n jenkins@${VM_IP} "umask 0022 && git clone ${ghprbAuthorRepoGitUrl} antrea && cd antrea && git checkout ${GIT_BRANCH} && DOCKER_REGISTRY=${DOCKER_REGISTRY} ./build/images/ovs/build.sh --pull && NO_PULL=${NO_PULL} make docker-test-integration"
-      if [[ "$COVERAGE" == true ]]; then
-        run_codecov "integration-tests" "coverage-integration.txt" "" true ${VM_IP}
-      fi
+    echo "===== Run Integration tests ====="
+    # umask ensures that files are cloned with the correct permissions so that Docker caching can be leveraged
+    ${SSH_WITH_UTILS_KEY} -n jenkins@${VM_IP} "umask 0022 && git clone ${ghprbAuthorRepoGitUrl} antrea && cd antrea && git checkout ${GIT_BRANCH} && DOCKER_REGISTRY=${DOCKER_REGISTRY} ./build/images/ovs/build.sh --pull && NO_PULL=${NO_PULL} make docker-test-integration;cd multicluster && NO_LOCAL=true make test-integration"
```

Comment:
@jianjuns , it's not about removing MC integration. I append the multicluster integration here `cd multicluster && NO_LOCAL=true make test-integration`. Both MC and antrea integration tests use the flag `integration-tests` now. 
Because both tests generates a integration file named `coverage-integration.txt`, codecov uploader will search the antrea root directory recursively and upload two files in different folder in the end.

Reply:
The flag is used by codecov to group different types of tests: https://docs.codecov.com/docs/flags. Antrea defined a few flags here: https://github.com/antrea-io/antrea/blob/main/codecov.yaml#L14-L38

Reply:
Ok. How about changing the title to "Run multi-cluster integration tests with test-integration"?

Reply:
Updated.

Reply:
Done. The title is changed to `Run MC integration tests with test-integration` in commit due to the length limitation.

Reply:
I merged the PR. Better to use multi-cluster rather than MC in commit message to be clear.